src/software/utils/sfmColorHarmonize/colorHarmonizeEngineGlobal.cpp

```diff
@@ -427,7 +437,8 @@ bool ColorHarmonizationEngineGlobal::Process()
       }
     }
 
-    const std::string out_folder = (fs::path(_outputDirectory) / (vec_selectionMethod[ _selectionMethod ] + "_" + vec_harmonizeMethod[ harmonizeMethod ])).string();
+    const int selectionMethodIdx = static_cast<int>(_selectionMethod);
+    const std::string out_folder = (fs::path(_outputDirectory) / (vec_selectionMethod[ selectionMethodIdx ] + "_" + vec_harmonizeMethod[ harmonizeMethod ])).string();
```

Comment:
`vec_selectionMethod[ selectionMethodIdx ]` => enumToString

Reply:
remove all usages of vec_selectionMethod and vec_selectionMethod

Reply:
Done